docs/resources/gluerecord.md

```diff
@@ -0,0 +1,23 @@
+# Resource: livedns_record
```

Comment:
After authoring this, I've realized that a data record would be useful, but I think I'd need to add a get request in the glue record in domain.go to request specific glue records, rather than [list all.](https://github.com/go-gandi/go-gandi/blob/master/domain/domain.go#L91-L94)